influxql/parser.go

```diff
@@ -372,20 +374,20 @@ func (p *Parser) parseInt(min, max int) (int, error) {
 	return n, nil
 }
 
-// parseUInt32 parses a string and returns a 32-bit unsigned integer literal.
-func (p *Parser) parseUInt32() (uint32, error) {
+// parseUInt parses a string and returns a 64-bit unsigned integer.
+func (p *Parser) parseUInt(base int, bitSize int) (uint64, error) {
```

Comment:
@corylanou  better example http://play.golang.org/p/wBE0JaqFa4.  Parsing as 64 bit and casting to 32 bit could silently wrap around and delete the wrong series.  This will become moot for series IDs in the near future when they're changed to 64 bit but I think we should just leave the ability to parse any bit size in the parser.  It's one small simple function that looks just like the stdlib function.  We may need to parse smaller sizes for something else in the future.